src/commands/fun/latex.ts

```diff
@@ -0,0 +1,86 @@
+import { ApplicationCommandOptionData, CommandInteraction, MessageEmbed, MessageAttachment, BaseGuild } from 'discord.js';
+import fetch from 'node-fetch';
+import { createCanvas, loadImage } from 'canvas';
+import { Command } from '@lib/types/Command';
+
+export default class extends Command {
+
+	description = `Accepts a LaTeX expression and posts it as a rendered image.`;
+
+	options: ApplicationCommandOptionData[] = [
+		{
+			name: 'input',
+			description: 'The LaTeX expression to render',
+			type: 'STRING',
+			required: true
+		},
+		{
+			name: 'transparent',
+			description: 'Whether to render the image with a transparent background (default: false)',
+			type: 'BOOLEAN',
+			required: false
+		}
+	]
+
+	async tempRun(interaction: CommandInteraction): Promise<void> {
+		// input should never be null
+		const tex = interaction.options.getString('input') ?? '';
+		const requestURL = 'https://latex.codecogs.com/svg.json?' + tex;
+		const errorResponse = "Sorry, I couldn't render that LaTeX expression.";
```

Comment:
I'm not sure if the const variable is needed given its only used twice, but I'll let the rest of the team weigh in on this.

Reply:
> I'm not sure if the const variable is needed given its only used twice, but I'll let the rest of the team weigh in on this.

I think it's fine to leave it as a const variable.